pyproject.toml

```
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyDiscRip"
+version = "1.0"
```

Should use the same version used in code:
https://github.com/AkBKukU/pyDiscRip/blob/e152fbbba6aced982fb3aa770a05b5c99ce41ba1/handler/media/cd.py#L147
```suggestion
version = "0.1"
```